docker-entrypoint.sh

```diff
@@ -7,36 +7,45 @@ export CASSANDRA_DIR="${BUILD_DIR}/cassandra"
 
 jekyll --version
 
-# Make sure we have the latest commit of Cassandra trunk
-cd ${CASSANDRA_DIR}
+cd ${CASSANDRA_SITE_DIR}/src
+
+# Now make the docs for the latest version
+pushd ${CASSANDRA_DIR}
+git clean -xdff
 git checkout trunk
 git pull --rebase --prune
+popd
+make .build-doc .latest-doc-link
 
-# Now make the docs for the latest version
-cd ${CASSANDRA_SITE_DIR}/src
-make add-latest-doc
 
+# Build docs for each of the released versions on the 3.11 branch
+pushd ${CASSANDRA_DIR}
+versions="$(git tag -l "cassandra-3.11.[5-8]*")"
+popd
 
-# Make sure we have the latest commit of Cassandra 3.11
+for v in $versions ; do
+  pushd ${CASSANDRA_DIR}
+  git clean -xdff
+  git checkout $v
+  popd
+  make .build-doc
+done
```

Comment:
i can delete this now. (haven't generated docs for all the release tags in the following commit.)

Reply:
done.